src/displayapp/screens/Clock.h

```diff
@@ -42,12 +43,14 @@ namespace Pinetime {
         Controllers::Settings& settingsController;
         Controllers::HeartRateController& heartRateController;
         Controllers::MotionController& motionController;
+        Controllers::FS& fs;
```

Comment:
optional suggestion: we could rename it to `filesystem` to be more explicit

see comment (far) above

Reply:
Done